CONTRIBUTING.md

```diff
@@ -2,17 +2,33 @@
 
 - Ensure you have node version 14 installed (suggestion: v14.18.1).
 - Ensure if you are using Windows to use the Windows Subsystem for Linux (WSL).
-- Run `./bootstrap.sh` to install the dependencies, and get the repo ready to be developed on.
-- Run `yarn start` inside of the `code` directory to start the development server.
+- Run `yarn start` directory to run a basic test Storybook "sandbox".
```

Comment:
I can't do code suggestions for multiple lines here because GH thinks this is just deleting lines, but heres, a suggestion (just move one text up):

```md
- Run `yarn start` directory to run a basic test Storybook "sandbox".

The `yarn start` script will generate a React Vite TypeScript sandbox with a set of test stories inside it, as well as taking all steps required to get it running (building the various packages we need etc).

# Running against different sandbox templates

You can also pick a specific template to use as your sandbox by running `yarn task`, which will prompt you to make further choices about which template you want and which task you want to run. 
```

Reply:
Got it!